lib/barbeque/runner/hako.rb

```diff
@@ -0,0 +1,46 @@
+require 'open3'
+
+module Barbeque
+  module Runner
+    class Hako
+      # @param [Barbeque::DockerImage] docker_image
+      # @param [String] hako_dir
+      # @param [Hash] hako_env
+      # @param [String] yaml_prefix
```

Comment:
We put a repository with Gemfile including hako to `/hako`. And we have an extra repository including yamls and its symlink is placed in `/hako/apps`. So our yamls are placed in `/hako/apps/foo.yml` but `apps` part can be changed. As you know, actually our `apps` repository's name is not `apps`...